src/components/checkbox/checkbox.scss

```diff
@@ -461,3 +458,44 @@ md-checkbox {
     }
   }
 }
+
+// Underlying native input element.
+// Visually hidden but still able to respond to focus.
+.md-checkbox-input {
+  position: absolute;
+  width: 0;
+  height: 0;
+  margin: 0;
+  padding: 0;
+  opacity: 0;
```

Comment:
Setting both `height: 0` and `width: 0` will definitely cause the checkbox to be considered "hidden" in some browser + screen-reader combinations. You should use the `md-visually-hidden` mixin.